build/esbuild-build.ts

```diff
@@ -14,13 +14,13 @@ export const entries = [...typescriptEntries, ...cssEntries];
 const allNetworkUrls: Record<string, string[]> = {};
 // this flattens all the rpcs into a single object, with key names that match the networkIds. The arrays are just of URLs per network ID.
 
-const blacklist = ["https://xdai-archive.blockscout.com"];
+const blacklist = ["https://xdai-archive.blockscout.com", "https://gnosis.api.onfinality.io/public"];
```

Comment:
This blacklist idea doesn't seem like a robust approach. We can merge it in but it should be handled dynamically. I think I implemented this in an unstable manner on my branch, I vaguely recall.

Reply:
It's because the CSS/style logic is implanted poorly. I tore it apart in my refactor but didn't get to put it all together in a totally stable way (mostly the other RPC related functions- I believe the styling code was significantly simplified and stable if I recall correctly).